camel/models/model_factory.py

```diff
@@ -0,0 +1,55 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from typing import Any, Dict
+
+from camel.models.base_model import BaseModel
+from camel.models.openai_model import OpenAIModel
+from camel.models.stub_model import StubModel
```

Comment:
```suggestion
from camel.models import BaseModel
from camel.models import OpenAIModel
from camel.models import StubModel
```
We can keep consistant import style with other subpackages

Reply:
Ok